Recurly/Response.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using RestSharp;
+
+namespace Recurly
+{
+    public class Response
+    {
+        public string RawResponse { get; set; }
+
+        public HttpStatusCode StatusCode { get; set; }
+
+        public IList<Parameter> Headers { get; set; }
```

Comment:
We decided here that we didn't want to expose `Parameter` which is from an internal dependency `RestSharp`